frontend/src/generated/graphql.ts

```diff
@@ -7524,6 +7524,46 @@ export type SampleHasMetadataSampleMetadataNodeAggregationWhereInput = {
   baitSet_SHORTEST_GTE?: InputMaybe<Scalars["Int"]>;
   baitSet_SHORTEST_LT?: InputMaybe<Scalars["Int"]>;
   baitSet_SHORTEST_LTE?: InputMaybe<Scalars["Int"]>;
+  cancerTypeDetailed_AVERAGE_EQUAL?: InputMaybe<Scalars["Float"]>;
```

Comment:
This change is a little surprising to me since we aren't adding these properties to the actual graph database. Are these here because they're part of the extended data types used by the custom resolvers?

Reply:
Good call-out. And yes. These changes don't do anything (we can't query these fields against Neo4j), but they get generated as long as these fields are part of our schema. If we want our generated GraphQL types to be extra clean we could run some scripts to remove them every time we re-run the codegen, but I don't think we can remove them from the type generation process entirely.

Reply:
Alternatively, we could potentially avoid this issue by using the [GraphQL CodeGen plugin](https://the-guild.dev/graphql/codegen/docs/custom-codegen/extend-schema) for extending the schema. I'll spend some time with this and let you know if it would be useful.

Reply:
I spent some time with extending the schema via the plugin, and it seems a bit buggy right now. I can make a separate ticket for this to spend more time on it if you think that's needed

Reply:
Something documenting the addition of these properties due to the custom resolvers should be sufficient. Let's plan to add/update docs on the dashboard models used.